### ISSUE TITLE
[Visual Requirement-Footer-save]Luminosity ratio is less than 4.5:1 for 'Pick a name' place holder on save edit field.

### ISSUE THREAD

**Issue by v-mesala** [Microsoft/pxt#2608]
**User Experience**
If the luminosity ratio is less than 4.5:1 for all the text content then it will be difficult for the visually impaired user to see what is present on the screen.

**Test Environment**
OS: RS2 Version 1703(OS Build 15063.483)
Platform: Edge.
Tool: Color contrast Analyzer

**Repro Steps**
1-Navigate to https://makecode.microbit.org/acc
2-Navigate to the Micro bit section element and select Code control given on it.
3-Navigate to the footer section and select 'Save' control and select it.
4-Launch Color Contrast Analyzer tool using WAT tool bar.
5-Verify, the luminosity ratio is greater than 4.5:1 for ‘pick a name’ placeholder or not. 

**Actual Result**
Luminosity ratio is less than 4.5:1 for ‘pick a name’ placeholder on save edit field.

**Expected Result**
Luminosity ratio should be greater than 4.5:1 for ‘pick a name’ placeholder on save edit field.

**MAS Reference**
https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

**Suggested fix**
1.1-Ensuring that a contrast ratio of at least 4.5:1 exists between text (and images of text) and background behind the text
1.2- If design allows , Not specify background color, not specify text color, and not using technology features that change those defaults .In this technique the author avoids having to do any contrast measures by simply not specifying the text color and not specifying the background. As a result the colors of both are completely determined by the user agent.
Tips:-
1.The easiest fix is to increase the brightness of light colors and decrease the brightness of darker colors.
2.Exaggerate lightness differences between foreground and background colors, and avoid using colors of similar brightness adjacent to one another, even if they differ in saturation or hue.
3.During design Use the Color Contrast Analyzer to determine whether new colors have color contrast ratio of at least 4.5:1.

**Key References for Fix**
http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html
Please refer the attachment file

**Please refer the attachment file**
![mas1 4 3_save_improper luminosity ratio for pick a name place holder name](https://user-images.githubusercontent.com/30434417/28822287-570a133c-76d6-11e7-8863-72282c78fd9f.png)

**Comment by v-mesala** [comment]
Bug is fixed and verified .Hence Closed the bug.
